pocs/utils/google/storage.py

```diff
@@ -97,3 +99,247 @@ def upload(self, local_path, remote_path=None):
                 'Problem uploading file {}: {}'.format(local_path, err))
 
         return remote_path
+
+    def get_file_blob(self, blob_name):
+        """Returns an individual blob (meta info about file).
+
+        Note:
+            See `get_file_blobs` for examples.
+
+        Args:
+            blob_name (str): Full name of Blob to be fetched.
+
+        Returns:
+            google.cloud.storage.blob.Blob|None: Blob object or None.
+        """
+        return self.bucket.get_blob(blob_name)
+
+    def get_file_blobs(self, prefix, filter_ext=None, include_pointing=False):
+        """Get storage blobs (meta info about files) matching the prefix.
+
+        Note:
+            The prefix can be any pattern, e.g:
+                - 'PAN006'  # All images for unit
+                - 'Hd189733'  # All images for field
+                - 'PAN006/Hd189733/7bab97' # All images for unit, field, camera
+                - 'PAN006/*/7bab97' # All images for unit, camera
+                - 'PAN006/Hd189733/7bab97/20180327T071126/' # Specific observation
+
+        Args:
+            prefix (str): Path in storage to observation sequence, see note.
+            filter_ext (str, optional): Filter by filename extension, e.g. '.fz'.
+            include_pointing (bool, optional): Whether or not to include the pointing file,
+                default False.
+
+        Returns:
+            google.cloud.storage.blob.Blob: Blob object.
+        """
+        objs = list()
+        for blob in self.bucket.list_blobs(prefix=prefix):
+            if 'pointing' in blob.name and not include_pointing:
+                continue
+            elif filter_ext is not None and blob.name.endswith(filter_ext) is False:
+                continue
+            else:
+                objs.append(blob)
+
+        return sorted(objs, key=lambda x: x.name)
+
+    def download_file(self, remote_path, save_dir='.', force=False, callback=None):
```

Comment:
Perhaps the default for save_dir should be either $HOME/Downloads or $TMPDIR? The latter is probably better.